gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -107,14 +206,24 @@ def main(args):
         )
         logger.info("CADD Hail Table for GRCh38 created.")
 
+    if args.revel:
+        logger.info("Creating REVEL Hail Table for GRCh38...")
+
+        ht = create_revel_grch38_ht()
+        ht.write(
+            get_insilico_predictors(predictor="revel").path,
+            overwrite=args.overwrite,
+        )
+
```

Comment:
```suggestion
        logger.info("REVEL Hail Table for GRCh38 created.")
```
To be consistent with cadd